cross-chain/solana/programs/tbtc/src/lib.rs

```diff
@@ -0,0 +1,357 @@
+use anchor_lang::prelude::*;
+use anchor_spl::{
+    associated_token::AssociatedToken,
+    // mint,
+    token::{mint_to, TokenAccount, Mint as SplMint, MintTo as SplMintTo, Token as SplToken},
+};
+
+declare_id!("HksEtDgsXJV1BqcuhzbLRTmXp5gHgHJktieJCtQd3pG");
+
+#[program]
+pub mod tbtc {
+    use super::*;
+    
+    pub fn initialize(ctx: Context<Initialize>) -> Result<()> {
+        let tbtc = &mut ctx.accounts.tbtc; 
```

Comment:
May be nicer to use set_inner so you don't miss setting a particular field in your struct.

Reference: https://docs.rs/anchor-lang/latest/anchor_lang/accounts/account/struct.Account.html#method.set_inner

Reply:
This comment goes for any account you initialize btw.